xmake/rules/winsdk/mfc/xmake.lua

```diff
@@ -43,8 +43,8 @@ rule("win.sdk.mfc.static")
 -- define rule: the application with shared mfc libraries
 rule("win.sdk.mfc.shared_app")
 
-    -- add mfc base rule
-    add_deps("win.sdk.mfc.env")
+    -- add mfc base and subsystem rule
+    add_deps("win.sdk.mfc.env", "platform.windows.subsystem")
```

Comment:
We don't need to add it for each rule. platform.windows has been globally applied to the c/c++ rules.